### ISSUE TITLE
letter-spacing is not taken into account when calculating size

### ISSUE THREAD

**Issue by skand888** [DesertFoxNV/ngx-autosize-input#53]
`letter-spacing: 0.03125em;` is not taken into account when calculating style for rendering text. 

```
const { fontStyle, fontVariant, fontWeight, fontSize, fontFamily } = this.style;

// font string format: {normal, normal, 700, 20px, Roboto, "Helvetica Neue", sans-serif}
ctx.font = fontStyle + ' ' + fontVariant + ' ' + fontWeight + ' ' + fontSize + ' ' + fontFamily;
```

As the result, if input style sets `letter-spacing` (such as Angular 15 `body-1` typography level), text overflows the control.

**Comment by joshuawwright** [comment]
@skand888 Thanks I will look into it.

**Comment by joshuawwright** [comment]
@skand888 This has been fixed in version 15.0.0. Cheers!

**Comment by skand888** [comment]
Thanks a lot! It works well now.
